tickets/templates/tickets/home.html

```diff
@@ -15,24 +15,31 @@
 
             {% for ticket_type in ticket_types %}
 
-            <div class="box">
-                <div class="icon">
-                    <i class="{{ ticket_type.emoji }}"></i>
+                <div class="box">
+                    <div class="icon">
+                        <i class="{{ ticket_type.emoji }}"></i>
+                    </div>
+                    <h2>{{ ticket_type.name }}</h2>
+                    <p class="text-small">{{ ticket_type.description }}</p>
```

Comment:
I don't know why this is commented out in the repo, but it shows on the site. Is the latest `main` branch the one that's deployed on the live site?